content/07.supplemental.md

```diff
@@ -1,14 +1,14 @@
 ## Supplemental Information Titles and Legends
 
-![**OpenPBTA Project Workflow, Related to Figure 1.** Biospecimens and data were collected by CBTN and PNOC. Genomic sequencing and harmonization (orange boxes) were performed by the Kids First Data Resource Center (KFDRC). Analyses in the green boxes were performed by contributors of the OpenPBTA project. Output files are denoted in blue. Figure created with BioRender.com.](https://raw.githubusercontent.com/AlexsLemonade/OpenPBTA-analysis/489d729cde4318654a26b0a9ef0dee6d706352cd/figures/pngs/figureS1.png?sanitize=true){#fig:S1 tag="S1" width="7in"}
+![**OpenPBTA Project Workflow, Related to Figure 1.** Biospecimens and data were collected by CBTN and PNOC. Genomic sequencing and harmonization (orange boxes) were performed by the Kids First Data Resource Center (KFDRC). Analyses in the green boxes were performed by contributors of the OpenPBTA project. Output files are denoted in blue. Figure created with BioRender.com.](https://raw.githubusercontent.com/AlexsLemonade/OpenPBTA-analysis/7d0e1c8b60fe258d3d7a61645f3ba8ba57025d3c/figures/pngs/figureS1.png?sanitize=true){#fig:S1 tag="S1" width="7in"}
```

Comment:
```suggestion
![**OpenPBTA Project Workflow, Related to Figure 1.** Biospecimens and data were collected by CBTN and PNOC. Genomic sequencing and harmonization (orange boxes) were performed by the Kids First Data Resource Center (KFDRC). Analyses in the green boxes were performed by contributors of the OpenPBTA project. Output files are denoted in blue. Figure created with BioRender.com.](https://raw.githubusercontent.com/AlexsLemonade/OpenPBTA-analysis/master/figures/pngs/figureS1.png?sanitize=true){#fig:S1 tag="S1" width="7in"}
```

Reply:
are you suggesting to untie the commits from these? There was a note in manubot formatting that the commit was required if this was from another repo, but I can try this if you want to go that route.

Reply:
Ah, I missed that info, can you point me to it about needing the specific commit? That documentation will help me understand why this has to be the case.

Reply:
yes, no worries - I think @jaclyn-taroni was the one who initially reviewed when I had started adding the new figures once compiled: https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#figures

Reply:
Note: although this appears that it pertains to only SVG, it was required that I added the commit/sanitize bits in order for the pngs to render as well.

Reply:
Makes sense about provenance!